### DISCUSSION TITLE
Свойства игрока

### DISCUSSION THREAD

**Discussion by Kukushechka7-ezx** [Sokol-Games-GmbH/PixelCombats.PublicApi#81] [Ideas]
@yurkapotam, @kkohno 

Предлагаю добавить в изначальные свойства эти значения...
`float hearts { get; } // хп прямо сейчас`
`bool DamageOnFall { get; set; } // получение урона от удара`
`bool DamageOnPlayer { get; set; } // урон от игрока так же распространение идёт на себя`
`bool DamageOnBuild { get; set; } // урон от блоков`
`Vector3d Camera { get; } // положение и направление камеры в отдельной структуре`
Тут конечно можно всё это дело внести в отдельные классы

**Comment by kkohno** [comment]
обдумаем, может чтото добавим
